emissary-core/src/netdb/dht.rs

```diff
@@ -107,10 +102,46 @@ impl<R: Runtime> Dht<R> {
 
     /// Get today's UTC date.
     fn utc_date() -> String {
```

Comment:
```suggestion
    fn utc_date(timestamp: Duration) -> String {
```
and remove the call to `R::time_since_epoch()` below. Then you can add a test that some timestamp from today equals "20250411"

Reply:
I decided to use u64 seconds as an argument to prevent misuse of the function when passing a duration not from the unix epoch